install.sh

```diff
@@ -39,14 +61,42 @@ CONFIG_GEN_PATH=$PWD/gen
 
 pushd "${BUFSTREAM_CLOUD}"
 
-echo "Applying Terraform..."
- TF_VAR_generate_config_files_path="${CONFIG_GEN_PATH}" \
+echo "Initializing Terraform..."
+  TF_VAR_generate_config_files_path="${CONFIG_GEN_PATH}" \
   TF_VAR_bufstream_metadata="${BUFSTREAM_METADATA}" \
   terraform init \
   --var-file "${BUFSTREAM_TFVARS}" \
   --var "bufstream_k8s_namespace=${BUFSTREAM_NAMESPACE:-bufstream}"
 
- TF_VAR_generate_config_files_path="${CONFIG_GEN_PATH}" \
+if [[ "$1" == "cleanup" ]] ; then
+  read -p "cleanup will destroy all resources. are you sure? type y/yes to continue: " confirm && [[ $confirm == [yY] || $confirm == [yY][eE][sS] ]] || exit 1
+  echo "Cleaning up Terraform..."
+  case ${BUFSTREAM_CLOUD} in
+    gcp)
+      echo "gcp"
+      clean_up_gcp
+      ;;
+    aws)
+      echo "aws"
+      clean_up_aws
+      ;;
+  esac
+  TF_VAR_generate_config_files_path="${CONFIG_GEN_PATH}" \
+  TF_VAR_bufstream_metadata="${BUFSTREAM_METADATA}" \
+  terraform destroy \
+  --var-file "${BUFSTREAM_TFVARS}" \
+  --var "bufstream_k8s_namespace=${BUFSTREAM_NAMESPACE:-bufstream}" \
+  -auto-approve
+  if [[ "$BUFSTREAM_CLOUD" == "gcp" || "$BUFSTREAM_CLOUD" == "aws" ]]; then
+    printf "warning - the bucket created by this script has to be deleted manually. it will begin emptying its contents soon. you can manually delete it at any time."
```

Comment:
id be more specific - "a deletion policy with a 1/0 day lifecycle limit was been added to minimize deletion costs, a manual deletion [...]"